distributions/openhab/processing.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<featuresProcessing xmlns="http://karaf.apache.org/xmlns/features-processing/v1.0.0">
+
+    <bundleReplacements>
+        <bundle originalUri="mvn:org.ops4j.pax.logging/pax-logging-api/[0,2.0.11)" replacement="mvn:org.ops4j.pax.logging/pax-logging-api/2.0.11" mode="maven" />
```

Comment:
Why not use 2.0.12?

Reply:
I could swear 2.0.11 was fine two days ago ;-), lets go with 2.0.12 then!